### PR TITLE
Feat watch later

### PR DESCRIPTION
**Pull request by nakulsharma15**
Feat: Added watch later functionality
1. User can add a video he added in watch later.
2. User can also delete the video he added in watch later

### CODE REVIEW

File: src/Components/HorizontalVideoCard.jsx [hunk 1]
```diff
@@ -1,66 +1,21 @@
 import "./Styles/HorizontalVideoCard.css";
 import { useUserDetails } from "../Contexts/UserContext/UserContext";
-import axios from "axios";
-import { toast } from "react-hot-toast";
-import toastStyle from "../Utils/toastStyle";
+import { unLikeHandler } from "../Utils/handleLikeUnlike";
+import { deleteHistoryHandler } from "../Utils/handleHistory";
+import { removeFromWatchLater } from "../Utils/handleWatchLater";
```
Comment (rahulcs754) [inline src/Components/HorizontalVideoCard.jsx:5]: you can create one index js and export all utilities then you have to just right one line.

File: src/Utils/handleLikeUnlike.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        if (res.status === 200 || res.status === 201) {
+            const { likes } = res.data;
+            userDispatch({ type: "ADD_TO_LIKED", payload: likes })
+            toast("Video Liked!", {
+                icon: '❤️',
+                style: toastStyle
+            });
+        }
+    } catch (error) {
+        console.log(error);
```
Comment (vaibhav-mahalle) [inline src/Utils/handleLikeUnlike.js:26]: remove the console.log here.

File: src/Utils/handleLikeUnlike.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        if (res.status === 200 || res.status === 201) {
+            const { likes } = res.data;
+            userDispatch({ type: "REMOVE_FROM_LIKED", payload: likes });
+            toast("Video Unliked!", {
+                icon: '💔',
+                style: toastStyle
+            });
+        }
+    } catch (error) {
+        console.log(error);
```
Comment (vaibhav-mahalle) [inline src/Utils/handleLikeUnlike.js:50]: remove the console.log() from everywhere.

**Review by vaibhav-mahalle — COMMENTED** [review]
just minor suggestions, everything else looks good.

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and nice 
just small suggestion

File: src/Utils/handleLikeUnlike.js (hunk 2 above)
Comment (nakulsharma15) [reply src/Utils/handleLikeUnlike.js:26]: I'll remove this

**Pull request merged**